src/ConcreteOperations/isdisjoint.jl

```diff
@@ -744,12 +744,31 @@ function is_intersection_empty(hp::Union{Hyperplane{N}, Line2D{N}},
     return is_intersection_empty_helper_hyperplane(hp, X, witness)
 end
 
-# disambiguation
 function is_intersection_empty(hp1::Union{Hyperplane{N}, Line2D{N}},
                                hp2::Union{Hyperplane{N}, Line2D{N}},
                                witness::Bool=false
                               ) where {N<:Real}
-    return is_intersection_empty_helper_hyperplane(hp1, hp2, witness)
+    if isequivalent(hp1, hp2)
+        res = false
+        if witness
+            w = an_element(hp1)
+        end
+    else
+        cap = intersection(hp1, hp2)
+        res = cap isa EmptySet
```

Comment:
I first tried solving a linear system, but that throws an exception if there is no intersection.

This implementation is not really efficient, but at least it does not crash as in `master`...

Reply:
to check disjointness of Line2D there is [this approach](https://github.com/JuliaReach/LazySets.jl/blob/master/src/ConcreteOperations/intersection.jl#L86)

Reply:
for the n-dimensional hyperplane case, there is a full length explanation [here](https://math.stackexchange.com/questions/2360603/how-to-find-the-intersection-of-two-hyperplanes-in-n-dimensions)

Reply:
> to check disjointness of Line2D there is this approach

Yep, that will be called by `intersection` :sweat_smile: